src/test/java/uk/gov/hmcts/reform/blobrouter/services/report/ReportServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.hmcts.reform.blobrouter.services.report;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.data.ReportRepository;
+import uk.gov.hmcts.reform.blobrouter.data.model.EnvelopeSummary;
+import uk.gov.hmcts.reform.blobrouter.model.out.EnvelopeSummaryItem;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Status.DISPATCHED;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Status.REJECTED;
+import static uk.gov.hmcts.reform.blobrouter.util.DateTimeUtils.instant;
+import static uk.gov.hmcts.reform.blobrouter.util.DateTimeUtils.localDate;
+import static uk.gov.hmcts.reform.blobrouter.util.DateTimeUtils.localTime;
+
+@ExtendWith(MockitoExtension.class)
+class ReportServiceTest {
+    public static final String CONTAINER_1 = "cont1";
+    public static final String FILE_NAME_1 = "file1.zip";
+    public static final String FILE_NAME_2 = "file2.zip";
+    private ReportService reportService;
+
+    @Mock
+    private ReportRepository reportRepository;
+
+    @BeforeEach
+    void setUp() {
+        reportService = new ReportService(reportRepository);
+    }
+
+    @Test
+    void getDailyReport_should_convert_date_into_date_range() {
+        // given
+        LocalDate dt = LocalDate.of(2019, 1, 14);
+        Instant from = instant("2019-01-14 00:00:00");
+        Instant to = instant("2019-01-15 00:00:00");
+
+        // when
+        reportService.getDailyReport(dt);
+
+        // then
+        verify(reportRepository).getEnvelopeSummary(from, to);
+        verifyNoMoreInteractions(reportRepository);
+    }
+
+    @Test
+    void getDailyReport_should_convert_repo_result() {
+        // given
+        Instant inst1 = instant("2019-01-14 10:11:12");
+        Instant inst2 = instant("2019-01-15 11:12:13");
+        Instant inst3 = instant("2019-01-16 12:13:14");
```

Comment:
As these instants don't provide any context and are only referenced once, it would be ideal not to define them at all and use their values directly instead.